<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setNoAlarm</class>
 <widget class="QWidget" name="setNoAlarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,10,0,0,0,0">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Szűrési feltételek</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFilters" stretch="2,3,0,3,0,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1">
       <item>
        <widget class="QLabel" name="labelZone">
         <property name="text">
          <string>Zóna</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxZone"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutPlace">
       <item>
        <widget class="QCheckBox" name="checkBoxPlace">
         <property name="text">
          <string>Helyszin szerint</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayoutPlace">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonPlacePattern">
           <property name="text">
            <string>Minta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditPlacePattern"/>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonPlaceSelect">
           <property name="text">
            <string>Kiválasztás</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxPlaceSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutService">
       <item>
        <widget class="QCheckBox" name="checkBoxService">
         <property name="text">
          <string>Szolgáltatás szerint</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayoutService">
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonServicePattern">
           <property name="text">
            <string>Minta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditServicePattern"/>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonServiceSelect">
           <property name="text">
            <string>Kiválasztás</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxServiceSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutNode">
       <item>
        <widget class="QCheckBox" name="checkBoxNode">
         <property name="text">
          <string>Eszköz (node) szerint</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayoutNode">
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonNodePattern">
           <property name="text">
            <string>Minta</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditNodePattern"/>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonNodeSelect">
           <property name="text">
            <string>Kiválasztás</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxNodeSelect">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,0,1">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelTable">
         <property name="text">
          <string>Szolgáltatás példány találatok :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Szolgáltatás példány</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hely</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tiltás típusa</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>-tól</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>-ig</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Kiválasztás</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayoutSel">
         <item>
          <widget class="QPushButton" name="pushButtonFetch">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Lekérdez</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonAll">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Mind</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonNone">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Egyiksem</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="labelMsg">
         <property name="text">
          <string>Indoklás :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditMsg"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Beállítandó állapot</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>További szűrés, az aktuális állpotra</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkBoxOff">
         <property name="text">
          <string>Nincs tiltás</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxOn">
         <property name="text">
          <string>Riasztás permanensen tiltva</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxTo">
         <property name="text">
          <string>Riasztás tiltva egy megadott időpontig</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFrom">
         <property name="text">
          <string>Riasztás tiltva egy magdott időponttól</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxFromTo">
         <property name="text">
          <string>Riasztás tiltva megadott időpontok között.</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutType">
         <item>
          <widget class="QRadioButton" name="radioButtonOff">
           <property name="text">
            <string>Nincs tiltás</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonOn">
           <property name="text">
            <string>Risztás(ok) permanens tiltása</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonTo">
           <property name="text">
            <string>Tiltás(ok) megadott időpontig</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonFrom">
           <property name="text">
            <string>Tiltás(ok) megadott időponttól</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonFromTo">
           <property name="text">
            <string>Tiltás(ok) megadott időíntervallumban</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutData">
         <item>
          <widget class="QLabel" name="labelFrom">
           <property name="text">
            <string>Tiltás kezdetének az időpontja :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEditFrom">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelTo">
           <property name="text">
            <string>Tiltás végének az időpontja :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEditTo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2000</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="currentSection">
            <enum>QDateTimeEdit::HourSection</enum>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSet">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Beállít</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Becsuk</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonPlaceSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxPlaceSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonNodeSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxNodeSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>836</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonServiceSelect</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxServiceSelect</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>525</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
